pages/fellows/zbilodea.md

```diff
@@ -0,0 +1,33 @@
+layout: fellow
+pagetype: fellow
+shortname: zbilodea
+permalink: /fellows/zbilodea.html
+fellow-name: Zoë Bilodeau
+title: Zoë Bilodeau - IRIS-HEP Fellow
+active: True
+dates:
+  start: 2022-05-16
+  end: 2022-08-05
+photo: /assets/images/team/Zoë-Bilodeau.jpg
+institution: Skidmore College
+e-mail: zbilodea@skidmore.edu
+project_title: Fast Integration of Poisson Distributions for Dead Sensor Marginalization
+project_goal: >
+The XENON collaboration's XENONnT particle detector is a dual-phase time projection chamber (TPC) made with the goal of detecting dark matter. To do this, XENON measures how many photons are detected by each sensor in the TPC during a collision between a xenon particle and a particle passing through the machine. Calculating the correlations between the numbers of photons enables inference of the types of particles that are reacting with the xenon in the detector, so that they will be able to identify a dark matter particle if one passes through. These correlations can create high dimensional problems which are far too costly for the current software to compute in a reasonable amount of time. The goal of this project is to optimize the calculation of the Poisson probability distribution, which is used in computing the correlations, so that when the next set of data is collected, these calculations can be done quickly and not require a highly sophisticated computer.
```

Comment:
```suggestion
  The XENON collaboration's XENONnT particle detector is a dual-phase time projection chamber (TPC) made with the goal of detecting dark matter. To do this, XENON measures how many photons are detected by each sensor in the TPC during a collision between a xenon particle and a particle passing through the machine. Calculating the correlations between the numbers of photons enables inference of the types of particles that are reacting with the xenon in the detector, so that they will be able to identify a dark matter particle if one passes through. These correlations can create high dimensional problems which are far too costly for the current software to compute in a reasonable amount of time. The goal of this project is to optimize the calculation of the Poisson probability distribution, which is used in computing the correlations, so that when the next set of data is collected, these calculations can be done quickly and not require a highly sophisticated computer.
```